repository/kafka/operator/templates/bootstrap.yaml

```diff
@@ -104,7 +104,7 @@ data:
     echo "Appending custom configuration file to the server.properties..." | xargs -L 1 echo $(date +'[%Y-%m-%d %H:%M:%S,%3N]') $1
 
     # remove blacklisted properties that users cannot override
-    skip_properties=("broker.id" "listeners" "advertised.listeners" "advertised.host.name" "listener.security.protocol.map" "log.dirs")
+    skip_properties=("broker.id=" "listeners=" "advertised.listeners=" "advertised.host.name=" "listener.security.protocol.map=" "log.dirs=")
```

Comment:
This is rather ugly, why not add the `=` in the `grep` and `sed` commands instead?

Reply:
the reason was not to add any overhead in `grep` and `sed` logic and keep them clean. But I will add a comment there around the reasoning for `=` and move it there